tests/hotplug/memory.go

```diff
@@ -60,11 +60,7 @@ var _ = Describe("[sig-compute][Serial]Memory Hotplug", decorators.SigCompute, d
 	Context("A VM with memory liveUpdate enabled", func() {
 
 		createHotplugVM := func(guest *resource.Quantity, sockets *uint32, maxSockets uint32, opts ...libvmi.Option) (*v1.VirtualMachine, *v1.VirtualMachineInstance) {
-			vmiOpts := append(libnet.WithMasqueradeNetworking(), libvmi.WithResourceMemory(guest.String()))
-			vmiOpts = append(vmiOpts, opts...)
-
-			vmi := libvmifact.NewAlpineWithTestTooling(vmiOpts...)
-
+			vmi := libvmifact.NewAlpineWithTestTooling(libnet.WithMasqueradeNetworking(), libvmi.WithResourceMemory(guest.String()))
```

Comment:
This is not correct, `opts` are not added anymore.

Reply:
:man_facepalming:

Reply:
Indeed missed, but it is also worrying that it passed e2e tests.
If it passed the tests, what does it mean about the [test](https://github.com/kubevirt/kubevirt/blob/0145200b4213460164d1603cddb62d9bcd068d5a/tests/hotplug/memory.go#L149) that uses it?

Reply:
Unfortunately our CI does not run any presubmit lane with hugepages support at the moment, so bad luck.

Reply:
@dhiller @brianmcarey can we fix this?

Reply:
> @dhiller @brianmcarey can we fix this?

If we can run it as part of the current lanes (i.e. change the nodes to have hugepages), great.
But I am not in favor of running another lane as a presubmit.

Do we have a postsubmit job that runs this test? Has it popped up as an alarm that it failed?

Reply:
Yes, it's fixed already https://github.com/kubevirt/kubevirt/commit/f8915253592692ff8a8f7c48b5b9f2b10b731d2f

Reply:
I think we should also add a linter for unused function parameters.